geode-core/src/main/java/org/apache/geode/internal/cache/wan/parallel/ParallelGatewaySenderQueue.java

```diff
@@ -1590,7 +1621,13 @@ public void cleanUp() {
 
   @Override
   public void close() {
```

Comment:
The modification to this close appears to have broken the test that was removed.  Since none of the new code calls close, what is the new implementation doing for us with the cleanQueues flags?

Reply:
Close is modified, cause inconsistent behavior between parallel and serial gw sender (what is reported as a fault). In case of parallel gw sender, without this modification, content of queue is lost when persistent GW sender is restarted.

Reply:
This is updated, according to your proposal. I t seems that closing of regain causes some additional, problems, like events in diskStore, will never be deleted. So parallel and serial GW sender queue are updated not to close region, at stoping of GW sender.

Reply:
Deleted TC is returned, to prove nothing is broken.